<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmFramelessDialogBase</class>
 <widget class="QDialog" name="frmFramelessDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1061</width>
    <height>463</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>frmFramelessDialogBase</string>
  </property>
  <widget class="QWidget" name="windowFrame" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>531</width>
     <height>261</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#windowFrame{border:1px solid palette(highlight); border-radius:5px 5px 5px 5px; background-color:palette(Window);}</string>
   </property>
   <widget class="WindowDragger" name="windowTitle" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>531</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#windowTitle{
border: 0px none palette(base); 
border-top-left-radius:5px; 
border-top-right-radius:5px; 
background-color:#20293a; 
height:20px;
}</string>
    </property>
    <layout class="QHBoxLayout" name="windowTitlebar">
     <item>
      <widget class="QLabel" name="titleText">
       <property name="styleSheet">
        <string notr="true">padding-left:5px;
color: rgb(255, 255, 255);
font: 75 14pt &quot;微软雅黑&quot;;</string>
       </property>
       <property name="text">
        <string>提示</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="closeButton">
       <property name="maximumSize">
        <size>
         <width>35</width>
         <height>35</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">#closeButton{
  background-color:none;
  border:none;
  width:16px;
  height:16px;
  padding:4px;
  image:url(:/res/icon_window_close.png);
  border-top-right-radius: 5px;
  padding-right:5px;
}
#closeButton:hover{
  background-color:palette(alternate-base);
	background-color: rgb(24, 26, 29);
}
##closeButton:pressed{
  background-color:palette(highlight);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="windowContent" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>911</width>
      <height>221</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#windowContent{
  border: 0px none palette(base);
  border-radius:0px 0px 5px 5px;
	background-color: rgb(255, 255, 255);
}</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>WindowDragger</class>
   <extends>QWidget</extends>
   <header>windowdragger.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
